Grafana list beautifications

rowich opened the pull request:
number columns right bound

bassmaster187 commented on TeslaLogger/Grafana/Speed Consumption.json:
```diff
@@ -52,9 +52,9 @@
         {
           "$$hashKey": "object:671",
-          "alias": "Last Driven",
+          "alias": "Last Driven * * * *",
```
Multilanguage will not work anymore

bassmaster187 commented on TeslaLogger/Grafana/Speed Consumption.json:
```diff
@@ -114,7 +114,7 @@
         },
         {
           "$$hashKey": "object:3424",
-          "alias": "Ø Consumption kWh",
```
All Language files has to be altered

rowich commented:
let me check where to find the other languages.
I simply added some 'align: "right"' lines. Still failing to understand
completely how these dashboards work.

Am 26.03.2021 um 09:18 schrieb Christian P.:
>
> ***@***.**** commented on this pull request.
>
> ------------------------------------------------------------------------
>
> In TeslaLogger/Grafana/Speed Consumption.json
> <https://github.com/bassmaster187/TeslaLogger/pull/543#discussion_r602089900>:
>
> > @@ -52,9 +52,9 @@
>        "styles": [
>          {
>            "$$hashKey": "object:671",
> -          "alias": "Last Driven",
> +          "alias": "Last Driven * * * *",
>
> Multilanguage will not work anymore
>
> —let m
> You are receiving this because you authored the thread.
> Reply to this email directly, view it on GitHub
> <https://github.com/bassmaster187/TeslaLogger/pull/543#pullrequestreview-621874675>,
> or unsubscribe
> <https://github.com/notifications/unsubscribe-auth/AL4L3BHMVPRXVNDO72JQTCTTFQ7NDANCNFSM4Z23JP6Q>.
>

rowich commented:
Ahhh.... ok, the original text is a find to replace in the language
file? OK! Will take care for that also.

Am 26.03.2021 um 09:19 schrieb Christian P.:
>
> ***@***.**** commented on this pull request.
>
> ------------------------------------------------------------------------
>
> In TeslaLogger/Grafana/Speed Consumption.json
> <https://github.com/bassmaster187/TeslaLogger/pull/543#discussion_r602090530>:
>
> > @@ -114,7 +114,7 @@
>
>          },
>
>          {
>
>            "$$hashKey": "object:3424",
>
> -          "alias": "Ø Consumption kWh",
>
> All Language files has to be altered
>
> —
> You are receiving this because you authored the thread.
> Reply to this email directly, view it on GitHub
> <https://github.com/bassmaster187/TeslaLogger/pull/543#pullrequestreview-621875432>,
> or unsubscribe
> <https://github.com/notifications/unsubscribe-auth/AL4L3BGYKU4GQTBM4TNKL2LTFQ7RDANCNFSM4Z23JP6Q>.
>

bassmaster187 commented:
> Ahhh.... ok, the original text is a find to replace in the language file? OK! Will take care for that also. Am 26.03.2021 um 09:19 schrieb Christian P.:

Exactly.
I think we should migrate to the new Grafana table. Actually it should have more or less all old features and more.

My sugestion: make a pull request without changes in alias and we try to migrate to the new grafana tables. If everything will work, maybe we could use all the new features. As far as I know you can set min min-width of columns with the new table.

rowich commented:
ok, great. The "min column width" is exactly the 2nd issue I tried to
address. You get a fresh request later today.

I have issues with my "200% scale factor" on the built-in screen of my
surface book (works on the external 100% screen, looks like the auto
column width does not work properly - and this is independant of the
used browser) and I like to avoid line breaks in columns as this
destroys the lookalike of the whole table

Am 26.03.2021 um 09:51 schrieb Christian P.:
>
>     Ahhh.... ok, the original text is a find to replace in the
>     language file? OK! Will take care for that also. Am 26.03.2021 um
>     09:19 schrieb Christian P.:
>
> Exactly.
> I think we should migrate to the new Grafana table. Actually it should
> have more or less all old features and more.
>
> My sugestion: make a pull request without changes in alias and we try
> to migrate to the new grafana tables. If everything will work, maybe
> we could use all the new features. As far as I know you can set min
> min-width of columns with the new table.
>
> —
> You are receiving this because you authored the thread.
> Reply to this email directly, view it on GitHub
> <https://github.com/bassmaster187/TeslaLogger/pull/543#issuecomment-808047079>,
> or unsubscribe
> <https://github.com/notifications/unsubscribe-auth/AL4L3BD4FAHH6IKON5FTHWLTFRDKPANCNFSM4Z23JP6Q>.
>

rowich commented:
I don't get it. I reverted the changes back to original and added the
following lines to "language-de.txt", but the "Speed Consumption"
dashboard still shows "Last Driven". Also made a reboot to ensure it
gets loaded. What is my misunderstanding of how the translation works?

#Speed Consumption.json
Last Driven=Letzte Fahrt  * * *
Ø Outside Temp C°=Ø Aussen °C
Ø Consumption kWh=Ø kWh
Speed km/h=Geschwindigkeit
Count=Anzahl

(I know, adding these "*" is not a perfect solution, I don't see another
possibility to anforce a bigger column width - any other idea is highly
appreciated. This is only an issue if working on a screen with a
scale-factor of 200% as on Surface Books. Grafana simply works as
expected on a scale factor of 100%)

Am 26.03.2021 um 09:19 schrieb Christian P.:
>
> ***@***.**** commented on this pull request.
>
> ------------------------------------------------------------------------
>
> In TeslaLogger/Grafana/Speed Consumption.json
> <https://github.com/bassmaster187/TeslaLogger/pull/543#discussion_r602090530>:
>
> > @@ -114,7 +114,7 @@
>
>          },
>
>          {
>
>            "$$hashKey": "object:3424",
>
> -          "alias": "Ø Consumption kWh",
>
> All Language files has to be altered
>
> —
> You are receiving this because you authored the thread.
> Reply to this email directly, view it on GitHub
> <https://github.com/bassmaster187/TeslaLogger/pull/543#pullrequestreview-621875432>,
> or unsubscribe
> <https://github.com/notifications/unsubscribe-auth/AL4L3BGYKU4GQTBM4TNKL2LTFQ7RDANCNFSM4Z23JP6Q>.
>
